scripts/checks

```diff
@@ -29,7 +29,13 @@ fi
 echo "+++ golint tree..."
 for i in $(find . -maxdepth 1 -type d -name '*' | grep -vE '/\..*$')
 do
-  golint "${i}"
+  output=$(golint "${i}") # golint has no exit codes
+  if [ -n "${output}" ]
+  then
+    echo -e "!!! golint checking failed: "
+    echo -n "${output}"
+    exit 1
```

Comment:
Yeah, that is a better idea. I will retool in a bit.

> On May 29, 2015, at 3:01 PM, Madhav Puri <notifications@github.com> wrote:
> 
> In scripts/checks <https://github.com/contiv/netplugin/pull/89#discussion_r31368691>:
> 
> > @@ -29,7 +29,13 @@ fi
> >  echo "+++ golint tree..."
> >  for i in $(find . -maxdepth 1 -type d -name '*' | grep -vE '/\..*$')
> >  do
> > -  golint "${i}"
> > +  output=$(golint "${i}") # golint has no exit codes
> > +  if [ -n "${output}" ]
> > +  then
> > +    echo -e "!!! golint checking failed: "
> > +    echo -n "${output}"
> > +    exit 1
> Just a minor nit. This will cause the script to error out on first module it hits errors, would it instead be better to collect all go-lint errors and print them together?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/89/files#r31368691>.
>